src/framework/audio/abstractsynthesizer.h

```diff
@@ -62,8 +62,8 @@ class AbstractSynthesizer : public IAudioSource, public async::Asyncable
     using EventsMapIteratorList = std::vector<EventsMapIterator>;
 
     struct EventsBuffer {
-        msecs_t from = 0;
-        msecs_t to = 0;
+        msecs_t m_from = 0;
```

Comment:
Can you explain what the warning was here?

Reply:
```
2022-03-17T09:21:45.7262068Z D:\a\MuseScore\MuseScore\src\framework\audio\abstractsynthesizer.h(88): warning C4458: declaration of 'from' hides class member
2022-03-17T09:21:45.7262976Z D:\a\MuseScore\MuseScore\src\framework\audio\abstractsynthesizer.h(65): note: see declaration of 'mu::audio::synth::AbstractSynthesizer::EventsBuffer::from'
2022-03-17T09:21:45.7263996Z D:\a\MuseScore\MuseScore\src\framework\audio\abstractsynthesizer.h(88): warning C4458: declaration of 'to' hides class member
2022-03-17T09:21:45.7265016Z D:\a\MuseScore\MuseScore\src\framework\audio\abstractsynthesizer.h(66): note: see declaration of 'mu::audio::synth::AbstractSynthesizer::EventsBuffer::to'```

Reply:
It is better to rename parameters in findEventsRange instead of rename struct members
Maybe as rangeFrom/rangeTo

We use prefix m_ for class member, not for struct

Reply:
OK, I'll do that then

Reply:
done

Reply:
Maker the change much leaner too